modules/core/shared/src/main/scala/gem/enum/HorizonsType.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.syntax.eq._
+import cats.instances.string._
+
+/**
+ * Enumerated type for horizons non-sidereal target type.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class HorizonsType(
+  val tag: String,
+  val name: String
+)
+
+object HorizonsType {
+
+  /** @group Constructors */ case object Comet extends HorizonsType("Comet", "Comet")
+  /** @group Constructors */ case object AsteroidNew extends HorizonsType("AsteroidNew", "Asteroid New")
+  /** @group Constructors */ case object AsteroidOld extends HorizonsType("AsteroidOld", "Asteroid Old")
```

Comment:
These will eventually be shown to the user so `Asteroid (New Format)` or similar might be more helpful.